services/user.py

```diff
@@ -0,0 +1,47 @@
+from django.contrib.auth import get_user_model
+from django.db.models import QuerySet
+
+from db.models import User
+
+
+def create_user(
+    username: str,
+    password: str,
+    email: str = None,
+    first_name: str = "",
+    last_name: str = "",
+) -> QuerySet[User]:
+    return get_user_model().objects.create_user(
+        username=username,
+        password=password,
+        email=email,
+        first_name=first_name,
+        last_name=last_name
+    )
+
+
+def get_user(user_id: int) -> QuerySet[User]:
```

Comment:
The return type of `get_user` should be `User` instead of `QuerySet[User]` since it returns a single user instance.